maintenance/fixconfig/main.go

```diff
@@ -207,10 +250,25 @@ func main() {
 	}
 	f.Sync()
 
-	// copy file to replace original
+	// move file to replace original
 	f.Close()
 	err = os.Rename(f.Name(), *configPath)
 	if err != nil {
-		log.Fatalf("Failed to replace config with updated version: %v", err)
+		// fallback to copying the file instead
+		src, err := os.Open(f.Name())
+		if err != nil {
+			log.Fatalf("Failed to replace config with updated version: %v", err)
```

Comment:
use a different error message? all three fatalf here are the same

Reply:
:shrug: different errors at the end of the message, but this message is the relevant part to the user. I should just move this to a function.

Reply:
refactored